ext/bcmath/libbcmath/src/doaddsub.c

```diff
@@ -182,7 +171,51 @@ bc_num _bc_do_sub(bc_num n1, bc_num n2, size_t scale_min)
 	}
 
 	/* Now do the equal length scale and integer parts. */
-	for (count = 0; count < min_len + min_scale; count++) {
+	count = 0;
+	if (min_bytes > 8) {
```

Comment:
When it was exactly 8 bytes, I didn't include the equal sign because I thought the overhead would be large for parallel operations. Do you think the overhead is small?

Reply:
Hmm, we should measure that.

Reply:
Including equals was often slightly faster. (Almost unchanged)